### DISCUSSION TITLE
Pick a journal for CF roadmap

### DISCUSSION THREAD

**Discussion by erget** [cf-convention/discuss#354] [Comments and ideas for changing CF]
### Topic for discussion

@sadielbartholomew @efisher008 @feggleton 
This is for us to follow up on picking a journal. 
By end of October we should have shortlist with pros and cons. We should propose this to the community and if we have a clear winner put that one forward; otherwise we might use the upvote/downvote mechanism to decide amongst equals. I have an opinion here but won't be too pushy about it ;)

I massaged ChatGPT to do some initial analysis of the journals we spoke about in the meeting. Here are the results:
| **Journal Name**                        | **Impact Score/Metric**              | **Notes on Fit**                                                                                             |
|-----------------------------------------|--------------------------------------|--------------------------------------------------------------------------------------------------------------|
| **Earth System Science Data (ESSD)**    | Impact Factor: 11.2, CiteScore: 18 | Highly relevant due to its focus on Earth system data standards and conventions, good fit for our roadmap.        |
| **Geoscientific Model Development (GMD)**| Impact Factor: 8.9 | Good fit as CF Conventions are used in modeling, and GMD focuses on model development and evaluation.     |
| **Environmental Modelling & Software**  | Impact Factor: 6.0 (2023) | Strong fit for the integration of standards like CF in environmental modeling tools and software. However it is focused on modelling & software - might not be ideal.              |
| **Journal of Open Research Software (JORS)**| N/A (no impact factor)              | Relevant if the paper emphasizes the software and technical aspects most.                    |

Journals discussed but not included:
- **Nature**: Too general and broad for a technical paper on data conventions like CF.
- **Frontiers in Geosciences**: Broader focus on geosciences, not specialized in data standards or models.
- **Journal of Geophysical Research: ML and Computation**: New journal, more specialized in machine learning, not directly related to CF.
- **BAMS (Bulletin of the American Meteorological Society)**: Focuses on atmospheric sciences, not on metadata standards.
- **EOS**: Primarily a magazine, not a research-focused journal, unsuitable for publishing technical roadmaps.

Instinctinvely I like ESSD, it's open access and seems to be a good fit - but I'm not sure as we're not publishing data, but rather a metadata standard. So it may not be such a great fit in the end.

**Comment by feggleton** [comment]
Journal Name | Impact Score/Metric | Notes on Fit| Publication Types | Submission to acceptance
-- | -- | --| -- | --
Earth System Science Data (ESSD) | Impact Factor: 11.2, CiteScore: 18 | Highly relevant due to its focus on Earth system data standards and conventions, good fit for our roadmap. | Data Description papers, Review articles, ESS Visions (open section to publicly discuss novel ideas, concepts, standards, or visions for the Earth System Science Data community as a whole), Brief communications. | 195 days
Geoscientific Model Development (GMD) | Impact Factor: 8.9 | Good fit as CF Conventions are used in modeling, and GMD focuses on model development and evaluation. | Model description papers, Development and technical papers, methods for assessment of models, model experiment description papers, review and perspective papers (summarize the status of knowledge and outline future directions of research within the scope of the journal.), corrigenda | 212 days
Environmental Modelling & Software (ScienceDirect)| Impact Factor: 6.0 (2023) | Strong fit for the integration of standards like CF in environmental modeling tools and software. However it is focused on modelling & software - might not be ideal. | Research Articles, Review papers, position papers, introductory overviews, commentaries, book reviews | 153 days
Journal of Open Research Software (JORS) | N/A (no impact factor) | Relevant if the paper emphasizes the software and technical aspects most. | Software Metapapers, Issues in Research Software | 328 days
Nature |Impact Factor: 50.5 (2023) | Too general and broad for a technical paper on data conventions like CF. | Research Articles, very little else for submissions. | 268 days
Frontiers in Earth Science - Solid Earth Geophysics | Impact factor: 2 CiteScore:3.5 | Broader focus, not specialized in data standards or models. | Original Research, Systematic Review, Methods, Review, Mini Review, Policy and practice reviews, Hypothesis and theory, Perspective, Data Report, Policy Brief, Brief Research Report, Opinion, Erratum, Correction, Editorial, Technology and Code |
Bulletin of the American Meteorological Society | 2023 Impact Factor: 6.9 | Focuses on atmospheric sciences, not on metadata standards. Covers quite a range of topics including Workshop summaries | Magazine - articles, essays |
EOS (AGU magazine) | |  Primarily a magazine, not a research-focused journal, unsuitable for publishing technical roadmaps. Proposals can include a maximum of 5 authors, including the submitter. No exceptions. Project teams or institutions cannot be authors. | Science update, opinion, feature article | 14-18 weeks. |
CODATA Data Science Journal | 5.4 - 28/133 in Computer Science (miscellaneous) (2023)|The CODATA Data Science Journal is a peer-reviewed, open access, electronic journal, publishing papers on the management, dissemination, use and reuse of research data and databases across all research domains, including science, technology, the humanities and the arts. The scope of the journal includes descriptions of data systems, their implementations and their publication, applications, infrastructures, software, legal, reproducibility and transparency issues, the availability and usability of complex datasets, and with a particular focus on the principles, policies and practices for open data. | Research articles, practice papers, review articles, essays | 262 days |

**Reply by efisher008** [reply]
Thank you @feggleton, this is a great start to scoping which journal would be most appropriate!

I'm looking at filling in some more information about the journals. However I can't find the ML and Computation section listed under Journal of Geophysical Research like we talked about in the meeting. From [this page](https://agupubs.onlinelibrary.wiley.com/journal/21562202), it looks as though they publish articles in the following sections:

- Atmospheres
- Biogeosciences
- Earth Surface
- Oceans
- Planets
- Solid Earth
- Space Physics

@sadielbartholomew do you have an idea where to find this?

**Reply by feggleton** [reply]
https://agupubs.onlinelibrary.wiley.com/journal/29935210 had to do a bit of digging to find it, think as it's new.

**Reply by feggleton** [reply]
Not much information on that one, so will leave it out of table for now.

**Reply by sadielbartholomew** [reply]
Thanks Fran, good find.

I am meant to be proactive on this topic to keep the momentum up on all of our behalves, as I volunteered at the CF Workshop - shall we arrange a meeting soon, to discuss?

**Reply by erget** [reply]
Hi folks, I've been running a bunch to catch up since I'm just getting back from the EUMETSAT Conference. Since we're all essentially in the same timezone I hope it won't be too hard to find an appointment. I really appreciate how you've kept the momentum going!

Previously we'd used the 14 UTC slot on Wednesdays for workshop planning - do we want to re-use that slot for our discussion next week? I've still got the placeholder so would be available then. If not, later in the week would also work for me.

**Reply by efisher008** [reply]
Yes, thanks for the link!

I think a meeting would be a good idea. Shall we sort out a time offline (e.g. via email) to meet?

**Reply by erget** [reply]
Correction, I've just seen that we'll probably stick with the slot for the workshop wrap-up. Would directly before or after that work for you? Thumbs up if yes, thumbs down if no :)

**Reply by efisher008** [reply]
Directly after the workshop wrap-up slot (i.e. 15 UTC / 16 BST / 17 CEST on 16th October) would work for me, but not the slot directly beforehand.

**Comment by erget** [comment]
@sadielbartholomew @efisher008 and I met today and reviewed the excellent work that @feggleton had posted previously. Thanks @feggleton your work made our lives easy.

We're not quite finished but have managed to throw all of the journals into [a sorting hat](https://docs.google.com/spreadsheets/d/1S9hOvSL5j9XxZAxkX5Fjev5aSxmLsbPNVBuZWjIMZBA/edit?usp=sharing) and make some sort of order out of it.

We've reviewed the journals based on their publication type; next week we'll review based on "Notes on fit" and also come to the next meeting having looked at the journals that we currently think are interesting candidates.

@efisher008 will set the next meeting to take place next week. Thumbs up if you want to be on the invite - see you there!

**Comment by sadielbartholomew** [comment]
Hi all, since our first meeting I have thought of another journal that might be a good choice, namely: [Scientific Data, which is a Nature publication](https://www.nature.com/sdata/journal-information):

> Scientific Data is a peer-reviewed open-access journal for descriptions of datasets and research that advances the sharing and reuse of research data

Because it is Nature based it will have a large reach but also it seems well-suited to our context (square brackets give my additions to indicate the relevance here):

> [although] Scientific Data primarily publishes Data Descriptors [which] provide detailed descriptions of research datasets[,]
Scientific Data also welcomes original articles on the technologies, policies, data standands, platforms and workflows used to support data sharing within a repository use case. "

**I think this is worth considering as an option, too - we have already discounted Nature for being "too general and broad", but a sub-publication like this could be more what we are looking for. It is still cross-domain, but focuses on considerations for research data in general including data standards which is exactly our scope.**

**Comment by larsbarring** [comment]
Hi all! 

Excellent work and evaluation of possible journals! 

I also like the addition of _[Nature] Scientific Data_ suggested by @sadielbartholomew. Nature is a high profile journal collection and I imagine that the threshold is higher than for other journals. Although their web site mainly focus on publication of _data [sets]_, there is at least one open call for a collection focussing on metadata [here](https://www.nature.com/collections/hbhbgcdcif). Having said this, I am in no way particularly favouring this journal over the other ones on the table.

**Comment by efisher008** [comment]
[like]  Fisher, Ellie (STFC,RAL,RALSP) reacted to your message:
________________________________

**Comment by erget** [comment]
The [sorting hat](https://docs.google.com/spreadsheets/d/1S9hOvSL5j9XxZAxkX5Fjev5aSxmLsbPNVBuZWjIMZBA/edit?usp=sharing) has spoken!
@efisher008 @feggleton @sadielbartholomew @erget met today and concluded that
1. Among the journals that we evaluated we've shortlisted Earth System Science Data (ESSD) and Bulletin of the American Meteorological Society (BAMS), that order. Both journals seem like good fits for the roadmap and have a lot of reach into the communities we want to reach.
2. We recommend submitting the roadmap to ESSD and, if rejected, pivoting to BAMS.
3. We furthermore recommend, if there's appetite to do this, publishing proceedings of the last CF meetings in BAMS, and considering publishing the proceedings of each year thereafter in a peer-reviewed journal.

Thoughts on this are welcome; we intend to present these recommendations at next Monday's meeting on producing a CF roadmap and closing this discussion if we come to an agreement there.

**Reply by erget** [reply]
Discussed with Roadmap Team. The Team likes all these recommendations. RE: (3) we could also consider Eos in addition to BAMS.
